src/openms/include/OpenMS/METADATA/PeptideHit.h

```diff
@@ -87,46 +87,18 @@ namespace OpenMS
     double mz = -1.;
     double intensity = 0.;
 
+    PeakAnnotation() = default;
```

Comment:
not needed

Reply:
It is. Sometimes it is not completely initialized.

Reply:
And bc of the other ctor